[INFO -- 2017-03-07 10:12:48]
Starting flip_value script
version: 2.0
[INFO -- 2017-03-07 10:12:48]
Initialization strings:
[INFO -- 2017-03-07 10:12:48]
(1) target extended-remote | ssh -T mic1 gdbserver --multi -
[INFO -- 2017-03-07 10:12:48]
(2) set sysroot /opt/mpss/3.7/sysroots/k1om-mpss-linux/
[INFO -- 2017-03-07 10:12:48]
(3) set sysroot /opt/mpss/3.7/sysroots/k1om-mpss-linux/
[INFO -- 2017-03-07 10:12:48]
(4) file /micNfs/gdb-fault-injection/injector/bin/dgemm/dgemm_check
[INFO -- 2017-03-07 10:12:48]
(5) set remote exec-file /micNfs/gdb-fault-injection/injector/bin/dgemm/dgemm_check
[INFO -- 2017-03-07 10:12:48]
(6) set args 228 2048 16 /micNfs/gdb-fault-injection/injector/bin/dgemm/dgemm_a_8192 /micNfs/gdb-fault-injection/injector/bin/dgemm/dgemm_b_8192 /micNfs/gdb-fault-injection/injector/bin/dgemm/gold_2048_m-order_228_ths_16_blocks 1
[INFO -- 2017-03-07 10:12:48]
initSignal:28
[INFO -- 2017-03-07 10:12:48]
endSignal:28
[INFO -- 2017-03-07 10:12:48]
seqSignal:15
[INFO -- 2017-03-07 10:12:48]
faultMode:1
[EXCEPTION -- 2017-03-07 10:13:23]
pythonException: invalid frame
[EXCEPTION -- 2017-03-07 10:13:23]
pythonException: invalid frame
[EXCEPTION -- 2017-03-07 10:13:23]
pythonException: invalid frame
[EXCEPTION -- 2017-03-07 10:13:23]
pythonException: invalid frame
[EXCEPTION -- 2017-03-07 10:13:23]
pythonException: invalid frame
[EXCEPTION -- 2017-03-07 10:13:23]
pythonException: invalid frame
[EXCEPTION -- 2017-03-07 10:13:23]
pythonException: invalid frame
[EXCEPTION -- 2017-03-07 10:13:23]
pythonException: invalid frame
[EXCEPTION -- 2017-03-07 10:13:23]
pythonException: invalid frame
[EXCEPTION -- 2017-03-07 10:13:23]
pythonException: invalid frame
[EXCEPTION -- 2017-03-07 10:13:23]
pythonException: invalid frame
[EXCEPTION -- 2017-03-07 10:13:23]
pythonException: invalid frame
[EXCEPTION -- 2017-03-07 10:13:23]
pythonException: invalid frame
[EXCEPTION -- 2017-03-07 10:13:23]
pythonException: invalid frame
[EXCEPTION -- 2017-03-07 10:13:23]
pythonException: invalid frame
[EXCEPTION -- 2017-03-07 10:13:23]
pythonException: invalid frame
[EXCEPTION -- 2017-03-07 10:13:23]
pythonException: invalid frame
[EXCEPTION -- 2017-03-07 10:13:23]
pythonException: invalid frame
[EXCEPTION -- 2017-03-07 10:13:23]
pythonException: invalid frame
[INFO -- 2017-03-07 10:13:23]
Backtrace BEGIN:
#0  0x000000000040375d in L_dgemm_185__par_region0_2_0 () at dgemm.c:221
#1  0x00007ffff790ba1f in ?? ()
#2  0x00007fffffffe550 in ?? ()
#3  0x0000000000000000 in ?? ()

216	                            CC_arr(i,j) = 0.0;
217	
218	                    for (kg=kk,k=0; kg<MIN(kk+block,order); k++,kg++)
219	                        for (jg=jj,j=0; jg<MIN(jj+block,order); j++,jg++)
220	                            for (ig=ii,i=0; ig<MIN(ii+block,order); i++,ig++)
221	                                CC_arr(i,j) += AA_arr(i,k)*BB_arr(j,k);
222	
223	                    for (jg=jj,j=0; jg<MIN(jj+block,order); j++,jg++)
224	                        for (ig=ii,i=0; ig<MIN(ii+block,order); i++,ig++)
225	                            C_arr(ig,jg) += CC_arr(i,j);

Backtrace END
Memory content before bitflip:0x020x000x000x00
Fault Mode: Double bit-flip
base address to flip value: 0x7fffffffdf40
address max offset: 4
Memory content after  bitflip:0x020x300x000x00
frame name: L_dgemm_185__par_region0_2_0
symbol name: block
symbol filename: dgemm.c
symbol line: 181
value: 12290
value address: 0x7fffffffdf40
Type: The type is an integer type.
Type sizeof: 4

[INFO -- 2017-03-07 10:13:23]
Fault Injection Successful after 35.129226923s
[INFO -- 2017-03-07 10:13:32]
event type: exit
[INFO -- 2017-03-07 10:13:32]
exit code: 0
